mkdocs.yml

```diff
@@ -1,20 +1,95 @@
+      - 'QC Alerts': 'features/analytics/dashboards/qcalerts.md'
+      - 'Social Shares': 'features/analytics/dashboards/socialshares.md'
+      - 'Referrer': 'features/analytics/dashboards/referrer.md'
+    - Progress: 'features/analytics/progress.md'
+  - Certificates: 'features/certificates.md'
+  - Learning Objectives: 'features/learningobjectives.md'
+  - Course Announcements: 'features/courseannouncements.md'
+  - External Tools: 
+    - CodeOcean: 'features/externaltools/codeocean.md'
+    - TeamBuilder: 'features/externaltools/teambuilder.md'
+- Best Practices:
+  - 'Getting Started: Design your Course': 'bestpractices/gettingstarted.md'
+  - 'Curriculum Design': 'bestpractices/curriculumdesign.md'
+  - 'Recommended Course Cycles':
+    - Course Cycles: 'bestpractices/recommendedcoursecycles/coursecycles.md'
+    - 'Scheduled': 
+      - Lecture: 'bestpractices/recommendedcoursecycles/scheduled/lecture.md'
+      - Hands-on: 'bestpractices/recommendedcoursecycles/scheduled/handson.md'
+      - Workshop: 'bestpractices/recommendedcoursecycles/scheduled/workshop.md'
+      - Nutshell: 'bestpractices/recommendedcoursecycles/scheduled/nutshell.md'
+    - 'Self-paced': 'bestpractices/recommendedcoursecycles/selfpaced.md'
+    - 'Blended': 'bestpractices/recommendedcoursecycles/blended.md'
+  - Prepare Material: 'bestpractices/preparematerial.md'
+  - Quiz Design: 'bestpractices/quizdesign.md'
+  - Community Management: 'bestpractices/communitymanagement.md'
+  - End of Course: 'bestpractices/endofcourse.md'
+  - Checklist: 'bestpractices/checklist.md'
+  - Project Management Gantt Chart: 'bestpractices/projectmanagement.md'
+- Course Administration:
+  - Setup Course Properties: 'courseadministration/courseproperties.md'
+  - Grant Permissions: 'courseadministration/permissions.md'
+  - Add Content: 
+    - Sections: 'courseadministration/addcontent/sections.md'
+    - Items: 
+      - Quiz: 'courseadministration/addcontent/items/quiz.md' 
+      - Video: 'courseadministration/addcontent/items/video.md' 
+      - Text  : 'courseadministration/addcontent/items/text.md' 
+      - LTI: 'courseadministration/addcontent/items/lti.md' 
+      - Peer Assessment: 'courseadministration/addcontent/items/peerassessment.md'
+  - Setup Certificate Templates: 'courseadministration/certificatetemplates.md'
+  - Setup Learning Objectives: 'courseadministration/learningobjectives.md'
+  - Quiz Management: 
+    - Grant Additional Attempts: 'courseadministration/quizmanagement/additionalattempts.md'
+    - Regrading: 'courseadministration//quizmanagement/regrading.md'  
+  - Peer Assessment Management: 
+    - Grant Additional Attempts: 'courseadministration/peerassessmentmanagement/additionalattempts.md'
+    - Conflicts: 'courseadministration/peerassessmentmanagement/conflicts.md'
+    - Regrading: 'courseadministration/peerassessmentmanagement/regrading.md'
+- Platform Administration:
```

Comment:
Do you really want to add that here? I thought this is something we want to move to our Confluence?